packages/rum-core/test/common/observers/page-clicks.spec.js

```diff
@@ -0,0 +1,142 @@
+/**
+ * MIT License
+ *
+ * Copyright (c) 2017-present, Elasticsearch BV
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ */
+
+import { createCustomEvent } from '../..'
+import { createServiceFactory, patchEventHandler } from '../../../src'
+import {
+  HISTORY,
+  TRANSACTION_SERVICE,
+  PERFORMANCE_MONITORING
+} from '../../../src/common/constants'
+import { observePageClicks } from '../../../src/common/observers'
+
+describe('observePageClicks', () => {
+  let serviceFactory
+  let transactionService
+  let performanceMonitoring
+  let unobservePageClicks
+  let containerElement
+
+  beforeEach(() => {
+    serviceFactory = createServiceFactory()
+    transactionService = serviceFactory.getService(TRANSACTION_SERVICE)
+    performanceMonitoring = serviceFactory.getService(PERFORMANCE_MONITORING)
```

Comment:
Before creating this PR, PERFORMANCE_MONITORING was "PerformanceMonitoring". I considered that it was a good time to create the constant for that and for 'ErrorLogging'.